### PR TITLE
Add release notes for 0.156

### PR DESCRIPTION
**Pull request by cberner**

### CODE REVIEW

File: presto-docs/src/main/sphinx/release/release-0.156.rst [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+* Add support for quantified comparison predicates: ``ALL``, ``ANY`` and ``SOME``.
+* Change default task concurrency to 16.
+* Add support for ``ARRAY`` and ``ROW`` in :func:`checksum` aggregation.
+* Improve performance of ``GROUP BY`` queries that compute a mix of distinct and non-distinct aggregations. This optimization
+  can be turned on by setting the ``optimizer.optimize-mixed-distinct-aggregations`` configuration option or via the ``optimize_mixed_distinct_aggregations``
+  session property.
+* Add support for filtered aggregations. E.g., ``SELECT sum(a) FILTER (WHERE b > 0) FROM ...``.
+* Add a variant of :func:``from_unixtime`` function that takes a timezone argument.
+* Fix query failure when using ``AT TIME ZONE`` in ``VALUES`` list
```
Comment (martint) [inline presto-docs/src/main/sphinx/release/release-0.156.rst]: Missing period

File: presto-docs/src/main/sphinx/release/release-0.156.rst [hunk 2]
```diff
@@ -0,0 +1,26 @@
+=============
+Release 0.156
+=============
+
+General Changes
+---------------
+
+* Add support for quantified comparison predicates: ``ALL``, ``ANY`` and ``SOME``.
+* Change default task concurrency to 16.
+* Add support for ``ARRAY`` and ``ROW`` in :func:`checksum` aggregation.
```
Comment (martint) [inline presto-docs/src/main/sphinx/release/release-0.156.rst]: Those types were already supported. More precisely, it's support for arrays and rows containing nulls.

File: presto-docs/src/main/sphinx/release/release-0.156.rst [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+* Improve performance of ``GROUP BY`` queries that compute a mix of distinct and non-distinct aggregations. This optimization
+  can be turned on by setting the ``optimizer.optimize-mixed-distinct-aggregations`` configuration option or via the ``optimize_mixed_distinct_aggregations``
+  session property.
+* Add support for filtered aggregations. E.g., ``SELECT sum(a) FILTER (WHERE b > 0) FROM ...``.
+* Add a variant of :func:``from_unixtime`` function that takes a timezone argument.
```
Comment (martint) [inline presto-docs/src/main/sphinx/release/release-0.156.rst]: Single backtick for :func:

File: presto-docs/src/main/sphinx/release/release-0.156.rst [hunk 4]
```diff
@@ -0,0 +1,26 @@
+=============
+Release 0.156
+=============
+
+General Changes
+---------------
+
+* Add support for quantified comparison predicates: ``ALL``, ``ANY`` and ``SOME``.
```
Comment (martint) [inline presto-docs/src/main/sphinx/release/release-0.156.rst]: I'd organize the items in this order:

* bug fixes (i.e., at timezone issue)
* new features (i.e., filtered aggs, quantified comparisons, new function, checksum improvements)
* performance improvements (i.e.. group by optimization, default task concurrency)

File: presto-docs/src/main/sphinx/release/release-0.156.rst [hunk 5]
```diff
@@ -0,0 +1,26 @@
...
+
+Hive Changes
+------------
+
+* Add support for legacy RCFile header version in new RCFile reader.
+
+SPI Changes
+-----------
+
+* Make ``ConnectorPageSink#finish()`` asynchronous.
```
Comment (electrum) [inline presto-docs/src/main/sphinx/release/release-0.156.rst]: Add the standard note that this is backwards incompatible.

File: presto-docs/src/main/sphinx/release/release-0.156.rst [hunk 6]
```diff
@@ -0,0 +1,26 @@
...
+Release 0.156
+=============
+
+General Changes
+---------------
+
+* Add support for quantified comparison predicates: ``ALL``, ``ANY`` and ``SOME``.
+* Change default task concurrency to 16.
+* Add support for ``ARRAY`` and ``ROW`` in :func:`checksum` aggregation.
+* Improve performance of ``GROUP BY`` queries that compute a mix of distinct and non-distinct aggregations. This optimization
```
Comment (electrum) [inline presto-docs/src/main/sphinx/release/release-0.156.rst]: This is too wide. Wrap to ~80 characters

**Comment by electrum** [comment]
Add a new section:
```
Redis Changes
-------------

* Support ``iso8601`` data format for the ``hash`` row decoder.

**Comment by martint** [comment]
One more item for the bug fixes section:

* Fix potential correctness issue in queries that contain correlated scalar aggregation subqueries.

File: presto-docs/src/main/sphinx/release/release-0.156.rst [hunk 7]
```diff
@@ -0,0 +1,41 @@
+=============
+Release 0.156
+=============
+
+General Changes
+---------------
+
+* Fix query failure when using ``AT TIME ZONE`` in ``VALUES`` list.
+* Add support for quantified comparison predicates: ``ALL``, ``ANY``, and
+  ``SOME``.
```
Comment (electrum) [inline presto-docs/src/main/sphinx/release/release-0.156.rst]: No need to wrap here. Makes it harder to read.

File: presto-docs/src/main/sphinx/release/release-0.156.rst [hunk 8]
```diff
@@ -0,0 +1,41 @@
...
+* Add support for :ref:`array_type` and :ref:`row_type` that contain ``NULL``
+  in :func:`checksum` aggregation.
+* Add support for filtered aggregations. E.g.,
+  ``SELECT sum(a) FILTER (WHERE b > 0) FROM ...``.
+* Add a variant of :func:`from_unixtime` function that takes a timezone
+  argument.
```
Comment (electrum) [inline presto-docs/src/main/sphinx/release/release-0.156.rst]: Don't wrap here

File: presto-docs/src/main/sphinx/release/release-0.156.rst [hunk 9]
```diff
@@ -0,0 +1,41 @@
...
+
+General Changes
+---------------
+
+* Fix query failure when using ``AT TIME ZONE`` in ``VALUES`` list.
+* Add support for quantified comparison predicates: ``ALL``, ``ANY``, and
+  ``SOME``.
+* Add support for :ref:`array_type` and :ref:`row_type` that contain ``NULL``
+  in :func:`checksum` aggregation.
+* Add support for filtered aggregations. E.g.,
```
Comment (electrum) [inline presto-docs/src/main/sphinx/release/release-0.156.rst]: Replace "e.g." with "For example: " and drop the period at the end

File: presto-docs/src/main/sphinx/release/release-0.156.rst (hunk 8 above)
Comment (cberner) [inline presto-docs/src/main/sphinx/release/release-0.156.rst]: You said 80 characters. It's 81

**Pull request merged**